client/src/App/index.js

```diff
@@ -1,10 +1,56 @@
 import React from 'react';
 import './style.css';
+import {
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import { ProvideAuth } from './use-auth';
+import { ProtectedRoute, AdminProtectedRoute } from './ProtectedRoute';
+import Navbar from '../Components/Common/Navbar';
 
 function App() {
   return (
-    <div>
-      <h1>Hello World !</h1>
+    <div className="App">
+      <ProvideAuth>
+        <Navbar />
+        <ul>
+          <li><Link to="/">Home</Link></li>
```

Comment:
Whenever you see repeated code, know you're doing something wrong.
Do not repeat code.
Instead of this, you can do something like this:
```jsx
[
  { to: '/', children: 'Home' },
  { to: '/cars', children: 'Cars' },
  ...
].map(link => <li key={link.to /* or preferrably an id */}><Link {...link} /></li>)
```